Taudem5PCVS2010/FlowdirConditioning/stdafx.cpp

```diff
@@ -0,0 +1,8 @@
+// stdafx.cpp : source file that includes just the standard includes
```

Comment:
This file appears to have been automatically generated. I prefer not to have it. See if we can configure project without it.